config.yaml

```diff
@@ -2,18 +2,22 @@
 # See LICENSE file for licensing details.
 
 options:
-  git-sync::repo:
+  git_sync_repo:
     type: string
     description: URL to repo to sync against
-  git-sync::branch:
+  git_sync_branch:
     type: string
     description: branch to sync
     default: main
-  git-sync::wait:
+  git_sync_wait:
     type: int
     description: git-sync wait interval between syncs (seconds).
     default: 60
-  prometheus_relpath:
+  prom_rules:
     type: string
     description: relative path in repo to prometheus rules
     default: ""
```

Comment:
Why no default? Please do add the default here, it is helpful for the admins to know how to shape new repos.

Reply:
If I provide a default then I would need to add another config option for enabling/disabling the usage of the rules.
For example if a repo has both loki and prom rules that happen to reside in the default path, but the user only wants to utilize loki rules from there, not prom rules.

Reply:
After refreshing my memory I withdraw the previous comment :)
I will set the default to match what the respective charm libs recommend, and the on/off mechanics is via add/remove relation.